test/src/components/traces/traces.spec.jsx

```diff
@@ -68,42 +74,51 @@ const pendingPromise = {
     case: ({pending}) => pending()
 };
 
-const stubResults = [{
-    traceId: '15b83d5f-64e1-4f69-b038-aaa23rfn23r',
-    root: {
-        url: 'test-1',
-        serviceName: 'test-1',
-        operationName: 'test-1'
-    },
-    services: [
-        {
-            name: 'abc-service',
-            duration: 89548,
-            spanCount: 12
-        },
-        {
-            name: 'def-service',
-            duration: 89548,
-            spanCount: 29
+const stubResults = [
+    {
+        traceId: '15b83d5f-64e1-4f69-b038-aaa23rfn23r',
+        root: {
+            url: 'test-1',
+            serviceName: 'test-1',
+            operationName: 'test-1'
         },
-        {
-            name: 'ghi-service',
-            duration: 89548,
-            spanCount: 31
-        }
-    ],
-    error: true,
-    startTime: 1499975993,
-    duration: 89548
-},
+        operationName: 'dummy',
+        rootError: false,
+        spanCount: 12,
+        serviceName: 'abc-service',
+        services: observable.array([
+            {
+                name: 'abc-service',
+                duration: 89548,
+                spanCount: 12
+            },
+            {
+                name: 'def-service',
+                duration: 89548,
+                spanCount: 29
+            },
+            {
+                name: 'ghi-service',
+                duration: 89548,
+                spanCount: 31
+            }
+        ]),
+        error: true,
+        startTime: 1499975993,
+        duration: 89548
+    },
     {
         traceId: '23g89z5f-64e1-4f69-b038-c123rc1c1r1',
         root: {
             url: 'test-2',
             serviceName: 'test-2',
             operationName: 'test-2'
         },
-        services: [
+        operationName: 'dummy',
+        rootError: false,
+        spanCount: 11,
+        serviceName: 'abc-service',
+        services: observable.array([
```

Comment:
made the services an observable array to match prop types of the real application.